client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -239,7 +223,6 @@ const reducer = combineReducers( {
 	siteVertical,
 	showSignupDialog,
 	plan,
-	selectedFeatures,
```

Comment:
The user selection of features should probably belong to Onboard store (same as selected vertical/fonts/design/domain, etc). In order to pass these to Site Setup flow we'll use site options (see related issue https://github.com/Automattic/wp-calypso/issues/45148) and they will become available [on the SITE_STORE](https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/hooks/use-site.ts#L18).